.github/workflows/deploy-firebase-acc.yml

```diff
@@ -18,14 +18,16 @@ jobs:
       - name: Configure
         uses: w9jds/firebase-action@master
         with:
-          args: functions:config:set clothingloop.base_domain="$BASE_DOMAIN" clothingloop.region="$REGION" clothingloop.admin_emails="$ADMIN_EMAILS"
+          args: functions:config:set clothingloop.base_domain="$BASE_DOMAIN" clothingloop.region="$REGION" clothingloop.admin_emails="$ADMIN_EMAILS" contactmail.to="$CONTACT_MAIL_TO" contactmail.cc="$CONTACT_MAIL_CC"
```

Comment:
Best to stick to the same naming conventions for configuration.

`contactmail.to` vs `clothingloop.admin_emails` and `clothingloop.base_domain`.

If you don't agree with a convention, refactor it all, don't mix things.